### PR TITLE
steam, steamcmd: add German translation

### PR DESCRIPTION
**Pull request by LukBukkit**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** Not known

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages.de/common/steamcmd.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# steamcmd
+
+> Ein Kommandozeilenwerkzeug, um über Steam verfügbare Anwendungen zu verwalten.
+> Weitere Informationen: <https://manned.org/steamcmd>.
+
+- Installiere und aktualisiere eine Anwendung ohne dich einzuloggen:
+
+`steamcmd +login anonymous +app_update {{anwendungs_id}} +quit`
```
Comment (kbdharun) [inline pages.de/common/steamcmd.md:8]:
```suggestion
`steamcmd +login {{anonymous}} +app_update {{anwendungs_id}} +quit`
```
Translate `{{anonymous}}` too

File: pages.de/common/steamcmd.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Installiere oder aktualisiere eine Anwendung unter Angabe deiner Zugangsdaten:
+
+`steamcmd +login {{benutzername}} +app_update {{anwendungs_id}} +quit`
+
+- Installiere eine Anwendung für eine bestimmte Plattform:
+
+`steamcmd +@sSteamCmdForcePlatformType {{windows}} +login {{benutzername}} +app_update {{anwendungs_id}} validate +quit`
```
Comment (kbdharun) [inline pages.de/common/steamcmd.md:16]: Update command argument for login to `anonymous`'s German equivalent.

File: pages.de/common/steam.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# steam
+
+> Eine Plattform für Videospiele von Valve.
+> Weitere Informationen: <https://developer.valvesoftware.com/wiki/Command_Line_Options#Steam_.28Windows.29>.
```
Comment (kbdharun) [inline pages.de/common/steam.md:4]:
```suggestion
> Weitere Informationen: <https://developer.valvesoftware.com/wiki/Command_Line_Options>.
```
I will create a new PR to make this change on other pages too.

File: pages.de/common/steamcmd.md (hunk 1 above)
Comment (LukBukkit) [reply pages.de/common/steamcmd.md:8]: The string `anonymous` is a constant required for the anonymous login, thus we can't translate it. Maybe we also should change this in the English version of the page by removing the braces. See https://developer.valvesoftware.com/wiki/SteamCMD#Anonymous

File: pages.de/common/steamcmd.md (hunk 2 above)
Comment (LukBukkit) [reply pages.de/common/steamcmd.md:16]: Here I would argue, that either we use `anonymous` as a constant (without the braces), or we use the concept of a login with an username established with the command before. Once we resolved the first discussion, we can apply the results to this command too 👍

File: pages.de/common/steamcmd.md (hunk 1 above)
Comment (kbdharun) [reply pages.de/common/steamcmd.md:8]: I guess we no need to translate `anonymous` as you suggested. Apply the curly braces in this suggestion alone and the PR is good to go.

**Review by kbdharun — APPROVED** [review]
LGTM, after the above suggestion, is resolved.

> Thanks for your contribution.

File: pages.de/common/steamcmd.md (hunk 1 above)
Comment (LukBukkit) [reply pages.de/common/steamcmd.md]: Thanks, I've applied your suggestion

**Pull request merged**